data/typelibrary/net-1.0.0/typelib/CLIENT_1.fbt

```diff
@@ -1,39 +1,40 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE FBType SYSTEM "http://www.holobloc.com/xml/LibraryElement.dtd">
-<FBType Comment="" Name="CLIENT_1">
-  <Identification Description="Copyright (c) 2017 fortiss GmbH&#13;&#10; &#13;&#10;This program and the accompanying materials are made&#13;&#10;available under the terms of the Eclipse Public License 2.0&#13;&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#13;&#10;&#13;&#10;SPDX-License-Identifier: EPL-2.0" Standard="61499-2"/>
-  <VersionInfo Author="Alois Zoitl" Date="2017-10-25" Organization="fortiss GmbH" Remarks="initial API and implementation and/or initial documentation" Version="1.0"/>
-  <InterfaceList>
-    <EventInputs>
-      <Event Comment="" Name="INIT" Type="Event">
-        <With Var="QI"/>
-        <With Var="ID"/>
-      </Event>
-      <Event Comment="" Name="REQ" Type="Event">
-        <With Var="QI"/>
-        <With Var="SD_1"/>
-      </Event>
-    </EventInputs>
-    <EventOutputs>
-      <Event Comment="" Name="INITO" Type="Event">
-        <With Var="QO"/>
-        <With Var="STATUS"/>
-      </Event>
-      <Event Comment="" Name="CNF" Type="Event">
-        <With Var="QO"/>
-        <With Var="STATUS"/>
-        <With Var="RD_1"/>
-      </Event>
-    </EventOutputs>
-    <InputVars>
-      <VarDeclaration Comment="" Name="QI" Type="BOOL"/>
-      <VarDeclaration Comment="" Name="ID" Type="WSTRING"/>
-      <VarDeclaration Comment="" Name="SD_1" Type="ANY"/>
-    </InputVars>
-    <OutputVars>
-      <VarDeclaration Comment="" Name="QO" Type="BOOL"/>
-      <VarDeclaration Comment="" Name="STATUS" Type="WSTRING"/>
-      <VarDeclaration Comment="" Name="RD_1" Type="ANY"/>
-    </OutputVars>
-  </InterfaceList>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is the whole file marked changed?

Reply:
because the 4diac IDE stores them in a different Format now .... 

if this is unwanted, i need to modify it with a Text Editor instead of 4diac IDE.

Reply:
ah now I know hat is going on. github is at least for me very bad to show me the differences.  We definitely want the new version. Sorry that I missed that.

Reply:
@azoitl i use Gitkraken to watch this, but it is bad as well. but in Gitkraken you can integrate Beyond-Compare: 

![](https://github.com/user-attachments/assets/c76a37e4-b26c-4e25-902a-41561e148f71)

looks like this then, 

and then you can turn "TRIV" on, to unmark trivial changes: 

then it looks like this: 

![](https://github.com/user-attachments/assets/778bbb56-c16f-4b84-875c-37aaeb6d83e3)

Reply:
If github would have shown me the differences between tabs and spaces I would have been already happy and understood it. Would see this if I go to EGit. But this is more hassle then just reading in github.